internal/curator/durable/state/fb/state.fbs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2017 Western Digital Corporation or its affiliates. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+namespace internal.curator.durable.state.fb;
+
+// TractIDF is a 10-byte value that can refer to a tract or an RS chunk, just like common.TractID.
+// Declare it with shorts so flatbuffers doesn't pad for excessive alignment.
+struct TractIDF {
+  b1: ushort;
+  b2: ushort;
+  b3: ushort;
+  b4: ushort;
+  b5: ushort;
+}
+
+// PartitionF holds data about one partition. ID is not strictly necessary since it should
+// always be determined by context, but it makes some code simpler, and there will be very
+// few of these.
+table PartitionF {
+  id: uint;
+  next_blob_key: uint;
+  next_rs_chunk_key: ulong; // 48 bits of "blob key" and "tract index" in one value
+}
+
+// TractF holds data about one tract of one blob.
+table TractF {
+  // Used for REPLICATED class.
+  hosts012: ulong;  // first three hosts packed into a ulong, 20 bits each
+  hosts3p: [uint];  // hosts beyond the first three
```

Comment:
"three plus"

Right, it's an optimization for the common case, which is expected to be replication factor 3. If we used `[uint]` for all the hosts, we'd use 12 bytes plus I think 4 or 6 more for the vector indirection and length. This way we use only 8 total since we don't have to store anything for hosts3p. But we can still support more if necessary.